File: testdata/add/initial/file.dart
void main() {
  print('Hello, World!');

  var greeting = Greeting('Hello');
  greeting.printMessage('World');
}

class Greeting {
  String message;

  Greeting(this.message);

  void printMessage(String name) {
    print('$message, $name!');
  }
}

File: testdata/add/initial/file.pl
#!/usr/bin/env perl
# Hello World in Perl

use strict;
use warnings;

print "Hello, World!\n";

# Define a subroutine
sub greet {
    my $name = shift;
    print "Hello, $name!\n";
}

# Call the subroutine
greet("Perl");